ARCHITECTURE.md

```diff
@@ -1,8 +1,8 @@
-# Gupaxx ARCHITECTURE
+# **Gupaxx ARCHITECTURE**
```

Comment:
No need to make header 1&2 bold.

Reply:
I disagree, I think that making the header of level 1 and 2 bold, creates a better readability.

if you still think that its better I will remove them.